eth/gaspricemonitor_test.go

```diff
@@ -59,23 +59,26 @@ func TestStart(t *testing.T) {
 
 	expErr := errors.New("SuggestGasPrice error")
 	gpo.SetErr(expErr)
-	err := gpm.Start(context.Background())
+	update, err := gpm.Start(context.Background())
+	assert.Nil(update)
 	assert.EqualError(err, expErr.Error())
 
 	// Switch back to no errors for SuggestGasPrice
 	gpo.SetErr(nil)
 
 	// Test success
 
```

Comment:
blank line can be removed I guess?

Reply:
There is a lot of variability when it comes to spacing of comments in the codebase - I believe some people prefer having the extra space for readability while some might prefer omitting the extra space to minimize LOC. Personally I don't mind either style and don't think it has any negative impact so I would suggest excluding those style nits during review - what do you think?

Reply:
Sounds good !